docs/release-notes/release_notes-5.1.3/master.adoc

```diff
@@ -0,0 +1,18 @@
+include::topics/templates/document-attributes.adoc[]
+:toc:
+:toclevels: 3
+:numbered:
+:imagesdir: topics/images
+:context: release-notes
+:release-notes:
+
+= {ReleaseNotesName}
+
+//Inclusive language statement
+include::topics/making-open-source-more-inclusive.adoc[]
+
+include::topics/rn-new-features-513.adoc[leveloffset=+1]
```

Comment:
Why are you creating new files for 5.1.3? In the past, we reused the files.

Reply:
I see what you're doing. Please don't create more folders. Just update the existing 5.1 release notes.